doc/manuals/orion-api.md

```diff
@@ -1608,21 +1706,43 @@ Those parameter are part of the URL request. They are mandatory.
 | `entityId` | string | Id of the entity to be retrieved      | `Room`     |
 | `attrName` | string | Name of the attribute to be retrieved | `Location` |
 
-**Request query parameters**
+_**Request query parameters**_
 
 | Parameter  | Optional | Type   | Description                                                                                 | Example       |
 |------------|----------|--------|---------------------------------------------------------------------------------------------|---------------|
 | `type`     | ✓        | string | Entity type, to avoid ambiguity in case there are several entities with the same entity id. | `Room`        |
 
-**Response**
+_**Request headers**_
+
+| Header               | Optional | Description                                                                                    | Example            |
+|----------------------|----------|------------------------------------------------------------------------------------------------|--------------------|
+| `Fiware-Service`     | ✓        | Tenant or service. See subsection [Multitency](#multitenancy) for more information.            | `acme`             |
+| `Fiware-ServicePath` | ✓        | Service path or subservice. See subsection [Service Path](#service-path) for more information. | `project`          |
+
+_**Response code**_
 
 * Successful operation uses 200 OK.
 * Errors use a non-2xx and (optionally) an error payload. See subsection on [Error Responses](#error-responses) for
   more details.
 
-Response 200 example:
+_**Response headers**_
+
+* If attribute value is JSON Array or Object:
+  * If `Accept` header can be expanded to `application/json` or `text/plain` return the value as a JSON with a
+    response type of application/json or text/plain (whichever is the first in `Accept` header or
+    `application/json` in case of `Accept: */*`).
+  * Else return a HTTP error "406 Not Acceptable: accepted MIME types: application/json, text/plain"
+* If attribute value is a string, number, null or boolean:
+  * If `Accept` header can be expanded to text/plain return the value as text. In case of a string, citation
+    marks are used at the beginning and end.
+  * Else return a HTTP error "406 Not Acceptable: accepted MIME types: text/plain"
+
+_**Response payload**_
 
-Content-Type is `application/json`
+The response payload can be an object, array, string, number, boolean or null. Further information about the complete attribute representation 
+described in the [JSON Attribute Representation](#json-attribute-representation) section.
```

Comment:
```suggestion
The response payload can be an object, array, string, number, boolean or null with the value of the attribute.
```

I think the reference to "JSON Attribute Representation" doesn't apply in this case, as `type` and `metadata` are not included in the response payload.